ktor-client/ktor-client-features/ktor-client-json/ktor-client-json-jvm/src/io/ktor/client/features/json/DefaultJvm.kt

```diff
@@ -1,6 +1,9 @@
 package io.ktor.client.features.json
 
+import io.ktor.client.call.TypeInfo
```

Comment:
Please consider using `*` imports like everywhere in ktor. It could be configured in IDEA kotlin code style